docs/source/index.rst

```diff
@@ -1,12 +1,10 @@
 BigchainDB Server Documentation
 ===============================
 
-Table of Contents
------------------
-
 .. toctree::
    :maxdepth: 1
 
+   ← Back to All BigchainDB Docs <https://bigchaindb.readthedocs.io/en/latest/index.html>
```

Comment:
Okay I made two pull requests to add it in those repos as well: https://github.com/bigchaindb/bigchaindb-common/pull/32 and https://github.com/bigchaindb/bigchaindb-driver/pull/90